### ISSUE TITLE
Background Android - not working

### ISSUE THREAD

**Issue by rodrigosimoesrosa** [Polidea/react-native-ble-plx#812]
Hi everyone!

version: 1.0.3 (Not running latest version)
react-native: 0.62.2

I'm working on a project that main idea is to connect with a specific device, and send commands and receive responses. After getting all this data, I'm persisting into a local database. It works very well for IOS, however, it doesn't for Android. For Android, functions like receives and sends stopped when the app goes to the background or sleep mode.
I know that the way IOS and Android handle the background task is different. So I would like to confirm with you guys if exist any other solution for that.
I believe that this question is more in terms of the architecture of how to use this library.

Current app:

React-native (layer) => 

- Bluetooth state;
- Connection with the device;
- Device logic;
- Send commands, and receive responses;
- Manage the state of devices;
- Persisting write commands/read response into a database; (It is dispatched by device state changes); **** _doesn't work for Android, the app stop completely_

Android native (layer) => I'm doing nothing right now;

=============================================================
Proposed solution app:

React-native (layer) => Do nothing;

Android native (layer) => 
- Bluetooth state;
- Connection with the device;
- Device logic;
- Send commands, and receive responses;
- Manage the state of devices;
- Persisting write commands/read response into a database;

* I NEED to persist the send commands and the receive responses into a database.

In other words, I need to know the device's state (connected/disconnected) / also some business states. The only solution that I found was to reimplement all flow of Bluetooth in this native part. Does anyone have more solutions for that?

Let me know if it wasn't clear.
Thank you guys

**Comment by maxueda** [comment]
I am experiencing exactly the same problem.
